src/backend/aspen/database/models/base.py

```diff
@@ -7,7 +7,7 @@
     schema="aspen",
     naming_convention={
         "ix": "ix_%(column_0_label)s",
-        "uq": "uq_%(table_name)s_%(column_0_name)s",
+        "uq": "uq_%(table_name)s_%(column_0_N_name)s",
```

Comment:
This changes the name of unique constraints to include *all* column names in the constraint instead of just the first one. It reduces the number of conflicting index names when alembic auto-generates migrations